proto/scripts/swagger.py

```diff
@@ -96,7 +96,17 @@ def rename_refs(obj: Any):
                 spec["paths"][url][method]["operationId"] = cur_id[len(operationid_prefix) :]
 
     with open(patch, "r") as f:
-        merge_dict(spec, json.load(f))
+        # TODO(DET-10185) Swaggo supports types like
+        #
+        # //	@Success 200     {object} model.HealthCheck
+        #
+        # but then we generate Python/Typescript types like
+        #
+        # class model.HealthCheck
+        #
+        # This is a syntax error. This is a small hack to get around this just
+        # by removing the "model." prefix.
+        merge_dict(spec, json.loads(f.read().replace("model.", "")))
```

Comment:
What's going on here? I'm not familiar with `swagger.py`.

Reply:
We have a swagger spec that we create and host for our API documentation. Also we generate ```bindings.py``` and ```api.ts``` from this swagger spec.

```swagger.py```` is used to combine swagger specs since we generate two and apply some processing. The first comes from our grpc API and the second comes from our echo API generated by swag https://github.com/swaggo/swag

The process is here.

https://github.com/determined-ai/determined/blob/85bb3c8881bbd23c34e07359db096fd337e90a51/proto/Makefile#L54-L68